components/store/StoreCard.js

```diff
@@ -1,41 +1,56 @@
 import { FontAwesome, MaterialCommunityIcons } from '@expo/vector-icons';
 import React from 'react';
-import { TouchableOpacity, View } from 'react-native';
+import { View } from 'react-native';
 import Colors from '../../assets/Colors';
 import { InLineContainer } from '../../styled/shared';
+import {
+  SpaceAroundRowContainer,
+  SpaceBetweenRowContainer
+} from '../../styled/shared';
+import { StoreCardContainer, EBTStatusBar } from '../../styled/store';
 import { Body, Caption, Title } from '../BaseComponents';
+import StoreProductButton from './StoreProductButton';
 
 /**
  * @prop
  * */
 
-function StoreCard({ store, callBack }) {
+function StoreCard({ store, callBack, seeProduct }) {
   const { name, hours, address, distance, ebt } = store;
   return (
-    <View style={{ marginBottom: 10 }}>
-      <TouchableOpacity onPress={callBack}>
-        <Title color={Colors.activeText}>{name}</Title>
-        <Caption color={Colors.secondaryText}>Distance: {distance} mi </Caption>
-        <InLineContainer>
-          <MaterialCommunityIcons name="directions" size={16} />
-          <Body color={Colors.secondaryText}>{address}</Body>
-        </InLineContainer>
-        <InLineContainer>
-          <FontAwesome name="clock-o" size={12} />
-          <Body color={Colors.secondaryText}>{hours}</Body>
-        </InLineContainer>
-
-        {/* Temporary EBT status; TODO @tommypoa to replace with tag */}
-        <Body color={Colors.secondaryText}>
-          EBT:{' '}
-          {ebt ? (
-            <Body color={Colors.secondaryText}>Yes</Body>
-          ) : (
-            <Body color={Colors.secondaryText}>No</Body>
+    <StoreCardContainer>
+      <SpaceBetweenRowContainer>
+        <SpaceAroundRowContainer>
+          <Title color={Colors.activeText}>{name}</Title>
+          {ebt && (
+            <EBTStatusBar>
+              <FontAwesome name="check" size={12} color="#004E14" />
+              <Caption color={Colors.darkerGreen}> EBT</Caption>
+            </EBTStatusBar>
           )}
-        </Body>
-      </TouchableOpacity>
-    </View>
+        </SpaceAroundRowContainer>
+        {seeProduct && <StoreProductButton callBack={callBack} />}
+      </SpaceBetweenRowContainer>
+      <Caption color={Colors.secondaryText}>Distance: {distance} mi </Caption>
+      <InLineContainer>
+        <MaterialCommunityIcons name="directions" size={16} />
+        <Body color={Colors.secondaryText}>{address}</Body>
+      </InLineContainer>
+      <InLineContainer>
+        <FontAwesome name="clock-o" size={12} />
```

Comment:
@ `MaterialCommunityIcon` and `FontAwesome` — probably best to use the same font library so you can be consistent with style and size.